src/blocks/ImageBlock/Image.astro

```diff
@@ -37,6 +37,8 @@ const imageUnavailableMessage = t('image_unavailable');
           loading={ loading }
           srcset={ getImageSrcSet(image.url) }
           src={ getImageSrc(image.url) }
+          height={image.height}
```

Comment:
This solves the same issue as the aspect ratio inline styling on line 43. So one or the other should be enough. I did notice that the inline calculation is actually calculated instead of dumped as a string, so thats probably why there is reflow with aspect ratio in place.

Reply:
Ok just to be clear, on page load the value of the aspect ratio is not correct, but when javascript kicks in it will be set correctly, which is too late ofcourse. No idea why JS has influence on this but on page https://head-start.pages.dev/en/demos/text-image-block/ the first image first has an aspect ratio of `aspect-ratio:1.5658959537572255;` and after JS kicks in it has a value of `aspect-ratio: 1.5659 / 1;`. You can test it with JS disabled. 🤷‍♂️

Reply:
And I dont know if this is actually a problem since 1.5659 / 1 is kind of the same as 1.5659

Reply:
Soooo, the problem seems to be the absence of width 100% on the image in combination with `aspect-ratio`. The aspect ratio works if you add width 100% to the image.